keras/mixed_precision/loss_scale_optimizer.py

```diff
@@ -307,7 +313,7 @@ def update_if_not_finite_grads():
 
 
 # See LossScaleOptimizer docstring for why this is so big
-_DEFAULT_INITIAL_SCALE = 2**15
+_DEFAULT_INITIAL_SCALE = 2 ** 15
```

Comment:
The linter doesn't like this change. It wants no spaces around the `**`.

Reply:
Thanks @reedwm . Running shell/format.sh on my machine actually added those spaces in the first place. It seems the linter scripts are a bit finicky about the environment in which they run.